test/unit/action/abandon.test.js

```diff
@@ -1,17 +0,0 @@
-import Action from '../../../src/action'
-
-describe('abandon', function() {
-  it('will not abandon the same child twice', function() {
-    const one = new Action(n => n)
-    const two = new Action(n => n)
-    const parent = new Action(n => n)
-
-    parent.adopt(one)
-    parent.adopt(two)
-
-    parent.abandon(one)
-    parent.abandon(one)
-
-    expect(parent.children).toEqual([two])
-  })
-})
```

Comment:
Removed. These are covered by other test suites and dig too much into internals.